Dockerfile

```diff
@@ -1,25 +1,21 @@
-# Image from https://hub.docker.com (syntax: repo/image:version)
-FROM continuumio/miniconda3:latest
+FROM quay.io/bcdev/xcube-python-base:0.8.1
+
+ARG INSTALL_PLUGINS=1
 
 # Metadata
 LABEL maintainer="helge.dzierzon@brockmann-consult.de"
 LABEL name=xcube
-LABEL version=0.8.0.dev7
+LABEL version=0.9.0.dev0
```

Comment:
Would be nice if we can derive this directly from `xcube.version`.

Reply:
Yes. That'd be nice. Could actually work in the actions.  I write that down as an issue